packages/components/src/components/tab/readme.md

```diff
@@ -10,20 +11,20 @@
 | `isSelected` | `selected` | Indicates whether the current tab is "selected" | `boolean` | `false`     |
 | `label`      | `label`    | The displayed text of the Tab                   | `string`  | `undefined` |
 
+
 ## Dependencies
 
 ### Used by
 
-- [stencila-tab-list](../tabList)
+ - [stencila-tab-list](../tabList)
 
 ### Graph
-
 ```mermaid
 graph TD;
   stencila-tab-list --> stencila-tab
   style stencila-tab fill:#f9f,stroke:#333,stroke-width:4px
 ```
 
----
+----------------------------------------------
 
-_Built with [StencilJS](https://stenciljs.com/)_
+*Built with [StencilJS](https://stenciljs.com/)*
```

Comment:
I might have missed it but is it worth adding `readme.md` to `.prettierignore` to avoid churn when these get regenerated.